### ISSUE TITLE
Account id when starting pump.io with != 80

### ISSUE THREAD

**Issue by redirect11** [e14n/pump.io#1066]
Hi, i figured out that when starting pump.io on another port different from 80, it will save all new registered users with an id like "http://hostname:port/api/user/username/profile" instead of "acct:username@hostname". I would to know if this could be a problem for the federation functionalities.

**Comment by evanp** [comment]
Yeah, I'm not sure. Don't run public sites on ports besides 80 or 443.

**Comment by stevenroose** [comment]
This could be fixed by #1072. Your pump can run on any post, as long as you can set the required webfinger files on port 443 of the domain (webfinger a la RFC 7033 required HTTPS).
